core/templates/dev/head/domain/topic/TopicUpdateService.ts

```diff
@@ -29,7 +29,7 @@ require('domain/topic/topic-domain.constants.ajs.ts');
 
 angular.module('oppia').factory('TopicUpdateService', [
   'ChangeObjectFactory', 'UndoRedoService',
-  'CMD_ADD_SUBTOPIC', 'CMD_ADD_UNCATEGORIZED_SKILL_ID',
```

Comment:
If `CMD_ADD_UNCATEGORIZED_SKILL_ID` is not used anywhere can we delete it from frontend constants? (and maybe from backend as well?)

Reply:
It is used in the backend, but not the frontend. I'll remove it from frontend constants.
The addition of an uncategorized skill happens in the dashboard and not in the editor. So, it is not present in the frontend, but the backend does use it.

Reply:
Done